postgres/rds.tf

```
@@ -118,4 +136,4 @@ data "aws_iam_policy_document" "enhanced-monitoring" {
       identifiers = ["monitoring.rds.amazonaws.com"]
     }
   }
-}
+}
```

github complaining about lack of newline
```suggestion
}

```